x/leverage/types/keys.go

```diff
@@ -33,6 +34,12 @@ func KeyRegisteredToken(baseTokenDenom string) []byte {
 	return util.ConcatBytes(1, KeyPrefixRegisteredToken, []byte(baseTokenDenom))
 }
 
+// KeySpecialAssetPair returns a KVStore key for getting and setting a SpecialAssetPair.
+func KeySpecialAssetPair(collateral, borrow string) []byte {
+	// pairprefix | collateralDenom | 0x00 | borrowDenom | 0x00 for null-termination
+	return util.ConcatBytes(1, KeyPrefixSpecialAssetPair, []byte(collateral), []byte{0}, []byte(borrow))
+}
```

Comment:
Ah, come to think of it I should be storing these using `SetDec` with the key like that,

Reply:
Nevermind on `SetDec` as that would make the "get all" code more complicated than `store.MustLoadAll`

However, I think there still might be an advantage to keeping denom in key, in case we ever

- optimize the code to only get pairs applying to the borrower's collateral (using `pairprefix | collateralDenom | 0x00` as MustLoadAll prefix)
- or create a query or operation which asks for `A->B` collateral weight, rather than sorting through all existing pairs